vm/src/vm.rs

```diff
@@ -300,13 +303,28 @@ impl VirtualMachine {
         TryFromObject::try_from_object(self, repr)
     }
 
-    pub fn import(&self, module: &str) -> PyResult {
-        match self.get_attribute(self.builtins.clone(), "__import__") {
-            Ok(func) => self.invoke(func, vec![self.ctx.new_str(module.to_string())]),
-            Err(_) => Err(self.new_exception(
-                self.ctx.exceptions.import_error.clone(),
-                "__import__ not found".to_string(),
-            )),
+    pub fn import(&self, module: &str, from_list: &PyObjectRef) -> PyResult {
```

Comment:
It might be good to move this whole function to the `import.rs` file. But that is something we can do later on.